Documentation improvements (#87)
liteflow-labs/liteflow-js: docs/pages/starter-kit/index.mdx

---
title: 'Starter Kit'
---

> # Starter kit
> 
> The starter kit is an example of an application running on the Liteflow infrastructure to showcase the uses of the infrastructure and/or be used as a starter kit to launch a product fast.

Reviewer: ```suggestion
# Starter kit

The starter kit is fully fledged NFT marketplace running on the Liteflow infrastructure to showcase the uses of the infrastructure and/or be used as a starter kit to launch a product fast.
```

This application is fork-able on this repository: https://github.com/liteflow-labs/marketplace-template.

## Features included

The Marketplace template includes the following features:

- Open collections (User-generated content)
  - Mint ERC-721
  - Mint ERC-1155
  - Creator's royalties support
- Featured NFTs on the home page
- Search system
- Explore NFTs with filtering and sorting systems

> - Native currency support

Reviewer: ```suggestion
- Native currency and ERC20 support
```

Author: updated the marketplace template repo as well

- List NFTs on sale
  - Fixed price listing
  - Timed auction listing

> - Make offers on NFTs
>   - Open offer
>   - Timed auction offer

Reviewer: ```suggestion
- Bid on NFTs
  - Open bid
  - Timed auction bid
```

Author: not sure about this one, need to check with @EmmanuelDrouin but we were pushing on offers rather than bid to be more accurate with what's the market uses

Reviewer 2: @antho1404 keep offers, we are indeed trying to fix a naming issue we have for too long ;)

- Purchase NFTs
- User profile
  - Account
    - Profile edition
    - Wallet page
  - NFTs "on sale"
  - NFTs "owned"
  - NFTs "created"
  - Offers ("bids") management
  - Trades history
  - Listings ("offers") management
- User verification system (verified status)
- Report system
  - Report a user
  - Report an NFT
- Notifications
  - In-app notifications

>   - Email notifications

Reviewer: This is done by the infra
```suggestion
```

Author: This is from the readme of the template https://github.com/liteflow-labs/marketplace-template/blob/main/README.md
Also I would not be against putting this here and push to handle email notifications at the application level ;)
